modules/monitoring-granting-users-permission-to-monitor-user-defined-projects.adoc

```diff
@@ -11,7 +11,7 @@ Cluster administrators can grant developers and other users permission to monito
 
 * The *monitoring-rules-view* cluster role provides read access to `PrometheusRule` custom resources for a project.
 
-* The *monitoring-rules-edit* cluster role grants a user permission to create, modify, and deleting `PrometheusRule` custom resources for a project.
+* The *monitoring-rules-edit* cluster role grants a user permission to create, modify, and delete `PrometheusRule` custom resources for a project. It also grants users the ability to silence alerts.
```

Comment:
I'm fine with this wording as-is, but I do think it's worth considering changing the plural "users" to "a user" for consistency with the way the rest of the documentation is written around the new sentence. :slightly_smiling_face: